tests/e2e/specs/out-of-game/stats.spec.js

```diff
@@ -283,3 +283,11 @@ describe('Stats Page', () => {
     });
   });
 });
+
+describe('Usage stats', () => {
+  beforeEach(setup);
+
+  it('Displays the site usage stats in the stats usage page', () => {
```

Comment:
I have been trying to work out how to test the chart and I'm stumped. Chartjs renders the chart in a `<canvas>` element which is kindof a black box, so there's little to be done as far as reading from the DOM. Atm we have a test on the data sent from the backend, (which this should be combined with), and that might be the best we can do. @seriouslysean penny for your thoughts.